python/cuml/metrics/cluster/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+
+
+set(cython_sources
+    adjusted_rand_index.pyx
+    completeness_score.pyx
+    entropy.pyx
+    homogeneity_score.pyx
+    mutual_info_score.pyx
+    silhouette_score.pyx
+    utils.pyx
+    v_measure.pyx
+)
+
+set(linked_libraries cuml::${CUML_CPP_TARGET})
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}"
+)
+
+# TODO: Finding NumPy currently requires finding Development due to a bug in CMake. This bug was
+# fixed in https://gitlab.kitware.com/cmake/cmake/-/merge_requests/7410 and will be available in
+# CMake 3.24, so we can remove the Development component once we upgrade to CMake 3.24.
+find_package(Python REQUIRED COMPONENTS Development NumPy)
+set(targets_using_numpy
```

Comment:
I just did a quick scan of a handful of these and I didn't see any numpy logic in the pyx files. Is the dependency inherited (maybe from raft), or is this not really necessary?